src/components/Header/Header.js

```diff
@@ -37,11 +39,11 @@ export const Header = () => {
         });
         setPodcastItems(sortedItems);
     }
-    const [navSize, satnavSize] = useState("10rem");
```

Comment:
This is what I was trying to avoid before when stating changes in my PR, If you check the files changed on the previous PR you will see the header is included. This might cause us some conflicts later.

Reply:
Is ok @yokwejuste 
You can do your changes first then I merge for now mine should not be merge yet . Just that I was confused if it was taken already.